rosdep/vcpkg.yaml

```diff
@@ -24,3 +24,15 @@ suitesparse:
   windows:
     vcpkg:
       packages: [suitesparse]
+mavlink:
+  windows:
+    vcpkg:
+      packages: [mavlink]
```

Comment:
I didn't see a mavlink port in the vcpkg, should we add it later?

Reply:
Updated.